src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -1055,8 +1015,8 @@ public static Vector<T> LoadAligned<T>(T* source)
         [Intrinsic]
         [CLSCompliant(false)]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static Vector<T> LoadAlignedNonTemporal<T>(T* source)
-            where T : unmanaged => LoadAligned(source);
+        public static Vector<T> LoadAlignedNonTemporal<T>(T* source) => LoadAligned(source);
+#pragma warning restore CS8500 // This takes the address of, gets the size of, or declares a pointer to a managed type ('T')
```

Comment:
This is unrelated, yes?  It's fine, just confirming.

Reply:
Oh, nevermind, it is related, it was relying on unmanaged being a subset of struct.